libs/p11-kit/Makefile

```diff
@@ -23,13 +23,14 @@ PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
```

Comment:
Do we need nls for a tool like p11-kit?

Reply:
No idea. I only noticed that there seems to be support for it.

Reply:
Let's not add it then, it only takes space and adds unnecessary dependencies.

Reply:
Removed